dizoo/box2d/lunarlander/envs/lunarlander_env.py

```diff
@@ -1,12 +1,14 @@
-from typing import Any, List, Union, Optional
-import time
+import os
+from typing import Optional
+
 import gym
+import imageio
```

Comment:
move `import imageio` inside of related function, and add import check like [this](https://github.com/opendilab/DI-engine/blob/main/ding/utils/compression_helper.py#L46)